web/src/components/tasks/LicenseStatusReceipt.tsx

```diff
@@ -77,34 +73,35 @@ const LicenseStatusLookup: Record<LicenseStatus, string> = {
   REVOKED: Config.licenseSearchTask.revokedStatusText,
 };
 
+const pendingStatuses: Set<LicenseStatus> = new Set([
+  "PENDING",
+  "DRAFT",
+  "SUBMITTED",
+  "UNDER_INTERNAL_REVIEW",
+  "SPECIAL_REVIEW",
+  "PENDING_DEFICIENCIES",
+  "DEFICIENCIES_SUBMITTED",
+  "CHECKLIST_COMPLETED",
+  "APPROVED",
+  "PENDING_RENEWAL",
+  "PENDING_REINSTATEMENT",
+]);
+
+const isPending = (licenseStatus: LicenseStatus): boolean => {
+  return pendingStatuses.has(licenseStatus);
+};
+
 export const LicenseStatusReceipt = (props: Props): ReactElement => {
-  const [theme, setTheme] = useState<PermitTheme>(pendingPermitTheme);
+  const [permitColorScheme, setPermitColorTheme] = useState<PermitTheme>(pendingPermitTheme);
```

Comment:
I agree with Spencer, the variables should be `permitColorTheme` and `setPermitColorTheme`

Reply:
Or, [permitColorScheme, setPermitColorScheme] as you stated in the reply.